apis/s3/v1beta1/replicationConfiguration_types.go

```diff
@@ -124,8 +124,8 @@ type ReplicationRule struct {
 type DeleteMarkerReplication struct {
 	// Indicates whether to replicate delete markers.
 	// In the current implementation, Amazon S3 doesn't replicate the delete markers.
```

Comment:
Is this comment still valid?

Reply:
no - it is not valid.  This setting can be used to successfully enable the replication of delete markers when set to `Enabled`